ROI_Maker.py

```diff
@@ -48,9 +50,18 @@ def ROI_Maker(spectrum, energy, sub_regions='auto'):
         bi_fwhm_channel = int(region_size * (bi_fwhm - E0) / Eslope)
         bi_peak_channel = int((609.31 - E0) / Eslope)
         bi_right_peak = bi_peak_channel + compton_distance * bi_fwhm_channel
-        bi_right_ch = (bi_right_peak - fwhm_channel,
-                       bi_right_peak + fwhm_channel)
+        bi_right_ch = (bi_right_peak + fwhm_channel,
+                       bi_right_peak - fwhm_channel)
```

Comment:
As above, the `- fwhm_channel` should come before the `+`. This matters because `Gamma_Analysis` lines 197-198 use these values as indices for slicing an array, and you have to slice left to right. So flip these back, and also below for `cs_left_ch`. Thanks.